scss/_custom-styles.scss

```diff
@@ -89,7 +89,8 @@
 */
 @each $color, $value in $colors {
```

Comment:
$colors here: https://github.com/az-digital/arizona-bootstrap/blob/main/scss/_custom-variables.scss#L27

Reply:
Right. I don't want to change the colors, just needed to add importance to the border-color style.

Reply:
Totally, just adding for reference